packages/tooling/fast-tooling/src/data-utilities/registered-action.ts

```diff
@@ -0,0 +1,44 @@
+export interface RegisteredActionCallbackConfig {
+    /**
+     * The unique identifier
+     */
+    id: string;
+}
+
+export interface RegisteredActionConfig extends RegisteredActionCallbackConfig {
```

Comment:
Is there a name other than "Registered" that would work here? "Registered" seems more like the state of a thing relative to another thing than the thing itself.